lorien/InfiniteCanvas/Tools/CanvasTool.gd

```diff
@@ -19,6 +19,10 @@ func _ready():
 	_canvas = get_parent()
 	set_enabled(false)
 
+# -------------------------------------------------------------------------------------------------
+func tool_event(event) -> void:
+	pass
+
```

Comment:
This is what tools should use now to receive events. `_input` still works, of course, but tool_event should be preferred.

Reply:
Any reason why event has no type here? I assume it's always an InputEvent?

Reply:
Good catch! Type should be InputEvent, always. Fix incoming!